de-lib/src/main/java/org/iplantc/de/diskResource/client/GridView.java

```diff
@@ -267,4 +268,7 @@
     LiveGridCheckBoxSelectionModel getSelectionModel();
 
     void setSingleSelect();
+
+    void setColumnPreferences(Map<String, String> preferences);
```

Comment:
Please add javadoc comments describing purpose of new public method.

Reply:
👍